stories/module-analytics-components.stories.js

```diff
@@ -48,3 +60,135 @@ generateReportBasedWidgetStories( {
 	component: DashboardAllTrafficWidget,
 	wrapWidget: false,
 } );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Dashboard/Bounce Rate Widget',
+	data: dashboardBounceRateWidgetData,
+	options: dashboardBounceRateWidgetArgs,
+	component: DashboardBounceRateWidget,
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Page Dashboard/Bounce Rate Widget',
+	data: dashboardBounceRateWidgetData,
+	options: dashboardBounceRateWidgetArgs,
```

Comment:
This should use page dashboard-specific data like in the All Traffic Storybook data above - otherwise it's just a duplicate.

Reply:
Updated.